substrate-node/runtime/src/lib.rs

```diff
@@ -700,30 +695,40 @@ construct_runtime!(
         NodeBlock = opaque::Block,
         UncheckedExtrinsic = UncheckedExtrinsic
     {
-        System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
-        Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent},
-        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
-        ValidatorSet: substrate_validator_set::{Pallet, Call, Storage, Event<T>, Config<T>},
-        Session: pallet_session::{Pallet, Call, Storage, Event, Config<T>},
-        Aura: pallet_aura::{Pallet, Config<T>},
-        Grandpa: pallet_grandpa::{Pallet, Call, Storage, Config, Event},
-        TransactionPayment: pallet_transaction_payment::{Pallet, Storage, Event<T>},
-        Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>},
-        Authorship: pallet_authorship::{Pallet, Storage},
-        TfgridModule: pallet_tfgrid::{Pallet, Call, Storage, Event<T>, Config<T>},
-        SmartContractModule: pallet_smart_contract::{Pallet, Call, Config, Storage, Event<T>},
-        TFTBridgeModule: pallet_tft_bridge::{Pallet, Call, Config<T>, Storage, Event<T>},
-        TFTPriceModule: pallet_tft_price::{Pallet, Call, Storage, Config<T>, Event<T>},
-        Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>},
-        BurningModule: pallet_burning::{Pallet, Call, Storage, Event<T>},
-        TFKVStore: pallet_kvstore::{Pallet, Call, Storage, Event<T>},
-        Council: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>},
-        CouncilMembership: pallet_membership::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>},
-        RuntimeUpgrade: pallet_runtime_upgrade::{Pallet, Call},
-        Validator: pallet_validator::{Pallet, Call, Storage, Event<T>},
-        Dao: pallet_dao::{Pallet, Call, Storage, Event<T>},
-        Utility: pallet_utility::{Pallet, Call, Event},
-        Historical: pallet_session::historical::{Pallet},
+        // System support
```

Comment:
Could you also fix the go client? It has a list of modules errors (in utils.go) and we map from id to error there. Will need to remove the id of Sudo pallet.

Reply:
Sure, but honestly having a staticly types moduleErrors list is quite a hastle to maintain. This should actually come from the metadata.

Reply:
https://github.com/threefoldtech/tfchain/issues/778

Reply:
I agree it's a hastle. Does the metadata also show the error information?